app/src/main/java/com/kickstarter/libs/utils/extensions/PledgeDataExt.kt

```diff
@@ -0,0 +1,46 @@
+@file:JvmName("PledgeDataExt")
```

Comment:
nit: I don't think we need this annotation as the compiler is already generating this java class with this name under the hood. Might be good to keep it though for consistency/readability.